src/services/credentialService.ts

```diff
@@ -23,3 +30,7 @@ export const getVerifiableCredential = async (args: IGetVerifiableCredentialArgs
 export const deleteVerifiableCredential = async (args: IDeleteVerifiableCredentialArgs): Promise<boolean> => {
   return dataStoreDeleteVerifiableCredential({ hash: args.hash })
 }
+
+export const createVerifiableCredential = async (args: ICreateVerifiableCredentialArgs): Promise<VerifiableCredentialSP> => {
```

Comment:
We really should use the Veramo methods and not directly call the LD local method. In the future we also want to be able to issue JWT credentials, which ICredentialIssuer/Veramo already supports. The only difference is that you will have to add a param that denotes the VC type (lds for JsonLD and jwt for JWT)